tests/foreman/cli/test_errata.py

```diff
@@ -1615,44 +1633,31 @@ def test_downgrading_package_shows_errata_from_library(
 
     :CaseImportance: High
     """
+    (datetime.utcnow() - timedelta(minutes=2)).strftime(TIMESTAMP_FMT)
```

Comment:
the result of this statement isn't being stored. is there an assignment missing here?